doc/compile_Linux.md

```diff
@@ -24,6 +24,11 @@
     cmake ..
     make install
 
+    # If you get an error while compiling for an AMD GPU (e.g. make[2]: *** No rule to make target '/opt/AMDAPPSDK-3.0/lib/x86_64/libOpenCL.so',   needed by 'bin/libxmrstak_opencl_backend.so'. Stop.) execute the following and then compile again:
+
+    $ cd $AMDAPPSDKROOT/lib/x86_64
+    $ sudo ln -sf sdk/libOpenCL.so.1 libOpenCL.so
```

Comment:
I am not sure if this is correct. On my system it does not work until I installed the block chain driver with `--compute`. The `.so` from the sdk folder was not able to handle my gpu. ON which system do you tested this?